src/12_scopes.py

```diff
@@ -20,7 +20,7 @@ def outer():
 
     def inner():
         y = 999
```

Comment:
@taniamichelle, make sure you go back and read the provided documentation on global, local, and nonlocal variables again. There should be a 'global' keyword placed between lines 7 & 8.
![Screen Shot 2019-12-09 at 10 12 21 AM](https://user-images.githubusercontent.com/16110931/70447268-76ca9700-1a6c-11ea-8e2e-173849f943df.png)

Reply:
There should also be a "nonlocal" keyword used between lines 21 & 22 in your code: 
![Screen Shot 2019-12-09 at 10 13 29 AM](https://user-images.githubusercontent.com/16110931/70447423-aa0d2600-1a6c-11ea-9eef-de7ca45013ba.png)